pkg/lib/userexport/model_test.go

```diff
@@ -65,5 +65,40 @@ func TestRequest(t *testing.T) {
 		`, `invalid request body:
 /format: enum
   map[actual:unknown format expected:[ndjson csv]]`)
+
+		test(`
+{
+	"format": "csv",
+	"csv": {
+		"fields": []
+	}
+}
+		`, "")
+		test(`
+{
+	"format": "csv",
+	"csv": {
+		"fields": [{ "unknown_pointer": "/sub" }]
+	}
+}
+		`, `invalid request body:
+/csv/fields/0: required
+  map[actual:[unknown_pointer] expected:[pointer] missing:[pointer]]`)
+		test(`
+{
+	"format": "csv",
+	"csv": {
+		"fields": [{ "pointer": "/sub" }]
+	}
+}
+		`, "")
+		test(`
+{
+	"format": "csv",
+	"csv": {
+		"fields": [{ "pointer": "/sub", "field_name": "user_id" }]
+	}
+}
+		`, "")
```

Comment:
Add a test 

```
{
	"format": "csv",
	"csv": {
		"fields": [{ "pointer": "/foobar" }]
	}
}
```

The error should show /foobar is not a valid value of an enum.

The schema is not strict enough. You need to specify all possible pointers as enum.

Reply:
as discussed, some traverse pointer like "/identities/0/claims/email" cannot pass enum validation
so skipped this implementation